src/Stratis.Bitcoin.Features.RPC/Models/TransactionInfoModel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using NBitcoin;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Stratis.Bitcoin.Features.RPC.Models
+{
+    public class TransactionInfoModel
+    {
+        public TransactionInfoModel()
+        {
+
+        }
+
+        public TransactionInfoModel(JObject tx)
+        {
+            if (tx["blockheight"] != null)
+            {
+                this.BlockHeight = (int)tx["blockheight"];
+            }
+
+            this.BlockHash = (string)tx["blockhash"];
+            decimal amount = (decimal)tx["amount"];
+            this.Amount = new Money((long)(amount * Money.COIN));
+            this.Confirmations = (int)tx["confirmations"];
+            this.TxId = (string)tx["txid"];
+            this.Index = (int)tx["blockindex"];
+
+            if (tx["generated"] != null)
+            {
+                this.Generated = (bool)tx["generated"];
+            }
+            else
+            {
+                // Default to True for earlier versions, i.e. if not present
+                this.Generated = false;
+            }
+        }
+
+        /// <summary>
+        /// The transaction id.</summary>
+        [JsonProperty(Order = 1, PropertyName = "txid")]
+        public string TxId { get; set; }
+
+        /// <summary>
+        /// The index of the output in the transaction.
+        /// </summary>
+        [JsonProperty(PropertyName = "blockindex")]
+        public int Index { get; set; }
```

Comment:
Based on what this field is intended to represent, the property should be named `BlockIndex` as well